azure_translator/__init__.py

```diff
@@ -0,0 +1,89 @@
+"""Azure Translator module."""
+import xml.etree.ElementTree as ET
+
+import requests
+
+
+class Translator(object):
+    """
+    Translator class, connects to Azure Translator API:
+
+        http://docs.microsofttranslator.com/text-translate.html
+    """
+    DEFAULT_LANGUAGE = 'en'
+    TOKEN_API = 'https://api.cognitive.microsoft.com/sts/v1.0/issueToken'
+    TRANSLATE_API = 'https://api.microsofttranslator.com/v2/http.svc/Translate'
+
+
+    def __init__(self, api_key):
+        self.api_key = api_key
+
+    @staticmethod
+    def _api_error(request_error):
+        return AzureApiError(
+            unicode(request_error),
+            response=request_error.response,
+            request=request_error.request
+        )
+
+    def get_access_token(self):
```

Comment:
I'm not necessarily for, one could use this to just retrieve the token and do something with it.